frameworks/PHP/nette/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "nette/web-project",
+	"description": "Nette: Standard Web Project",
+	"keywords": ["nette"],
+	"type": "project",
+	"license": ["MIT", "BSD-3-Clause", "GPL-2.0", "GPL-3.0"],
```

Comment:
Is it correct all these licenses togheter ?

Reply:
With the `GPL` you can't create proprietary apps !!!

Reply:
Yes they actually let you choose your license. https://github.com/nette/application/blob/master/license.md